execution/gethexec/executionengine.go

```diff
@@ -414,13 +414,13 @@ func (s *ExecutionEngine) sequenceDelayedMessageWithBlockMutex(message *arbostyp
 		DelayedMessagesRead: delayedSeqNum + 1,
 	}
 
-	err = s.consensus.WriteMessageFromSequencer(lastMsg+1, messageWithMeta)
+	startTime := time.Now()
+	block, statedb, receipts, err := s.createBlockFromNextMessage(&messageWithMeta, false)
```

Comment:
Block is being created here before "writing the message" now.
I don't fully understand how startTime is being used down the line, so not sure if this change will negatively impact its usage.

Reply:
I don't fully understand the implications of changing the order of these, @tsahee or @PlasmaPower can probably comment.

Reply:
OK, I will do this :).
Not sure for what this compute block time is used though.

Reply:
Done.